src/middleware.js

```diff
@@ -1,54 +0,0 @@
-import morgan from 'morgan';
```

Comment:
There is no more middleware that `nodule-logging` exports. Instead the `logger` factory sets up a logging midleware on the app for you.